src/openmmio.py

```diff
@@ -1191,14 +1267,11 @@ def energy_rmsd(self, shot=0, optimize=True):
 
         self.update_simulation()
 
-        if self.platname in ['CUDA', 'OpenCL'] and self.precision in ['single', 'mixed']:
-            crit = 1e-4
-        else:
-            crit = 1e-6
+        crit = 1e-4 if (self.platname in ['CUDA', 'OpenCL'] and self.precision in ['single', 'mixed']) else 1e-6
```

Comment:
Same style comment as above

Reply:
I've reverted this in the latest commit.